web/services/wallet_services.py

```
@@ -0,0 +1,21 @@
+from eth_account import Account
+from mnemonic import Mnemonic  # New import for BIP-39 mnemonics
+
+
+def create_student_wallet():
```

Ruff (0.8.2)

Missing return type annotation for public function `create_student_wallet`

(ANN201)